src/controllers/workspace.controller.js

```diff
@@ -0,0 +1,23 @@
+const Workspace = require('../models/workspace.models');
+const {response_201, response_500} = require('../utils/responseCodes.utils')
+
+exports.createWorkspace = async (req, res) => {
+    const workspace = new Workspace({
+        name: req.body.name,
+        description: req.body.description,
+        workspaceId: req.body.workspaceId,
+        createdBy: req.body.userId,
+    });
+
+    try {
+        const savedWorkspace = await workspace.save();
+        return response_201(res, 'Workspace created successfully', {
+            name: savedWorkspace.name,
+            description: savedWorkspace.description,
+            workspaceId: savedWorkspace.workspaceId,
```

Comment:
Also make sure the workspaceId is not already taken. If taken give appropiative errors

Reply:
it will err if workspace it is not unique as defined in the model